opennebula/defaults.yaml

```diff
@@ -85,13 +66,7 @@ kernel:
     novnc:
       service: {}
     oneflow:
```

Comment:
Please remove this line since oneflow becomes a None-typed variable that can be merged with other dicts.